pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -14,6 +14,9 @@ boxmaker = fetchurl {
   sha256 = "5c5697f43dc3a95468f61f479cb50b7e2b93379a1729abf19e4040ac9f43a1a8";
 };
 
+inherit (pythonPackages) python numpy;
+pyxml = pkgs.pyxml.override{ inherit python; };
+
```

Comment:
In #13057 I moved pyxml into `pythonPackages.nix`. It should be sufficient now to write `inherit (pythonPackages) python lxml pyxml numpy;` in `all-packages.nix`. Of course those should be the arguments then of this function. No need to override.